.github/workflows/master.yml

```diff
@@ -24,9 +24,26 @@ jobs:
       run: |
         echo "app_ver=$(mvn -B help:evaluate -Dexpression=project.version -q -DforceStdout)"
         echo "app_ver=$(mvn -B help:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
+
+    -
+      name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    -
+      name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+# Docker hub action
+    -
+      name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
     -
       name: Build with Maven
-      run: mvn -B clean integration-test package assembly:single docker:build
+      run: mvn -B clean integration-test package assembly:single docker:build docker:push
```

Comment:
This pushes snapshot builds to Docker Hub. Why is that?

Reply:
Hi. I think that in current master, it should also push snapshots and release builds are additionally tagged with :latest.

I think the reason why this is included now within the same build command is due to buildx - it builds the container in an embedded build environment (to emulate arm64) and since it's also wrapped inside docker-maven-plugin, there is no good way to get the built image back to running environment. So buildx is asked to directly push the image.

Reply:
That makes sense. It also makes it easier for people that want to test with a snapshot build.
@davideicardi Do you agree?

Reply:
Yes, it looks good to me. Do you think that we need to change something for release build?

Reply:
I don't think further changes are needed. The lines below take care of the release process and they still do what happened before: apply a tag of the release and add the `latest` tag.
I'm going to merge this PR.